types/interfaces.ts

```diff
@@ -16,7 +16,7 @@ export interface EditorFunction {
 export interface EditorLanguages {
   [language: string]: {
     program: string
-    defaultCode?: string
+    defaultCode?: string | string[]
```

Comment:
I went ahead and made it possible to be a string for a single defaultCode or an array for multiple editors on one page.